src/private/functions.php

```diff
@@ -127,6 +126,85 @@ function _util_uri_apply_rules($uriPath, $incoming)
     return implode('/', $fragments);
 }
 
+/**
```

Comment:
Moved here because we do not want dependencies between `src/private/functions.php` and `src/DDTrace`.